lib/fake_stripe/stub_stripe_connect.rb

```diff
@@ -0,0 +1,37 @@
+require "fake_stripe/bootable"
+require "sinatra/base"
+
+module FakeStripe
+  class StubStripeConnect < Sinatra::Base
+    extend Bootable
+
+    CODE = "stripe_connect_express_oauth_authorization_code".freeze
+
+    post "/oauth/token" do
+      code = params[:code]
+      if code == CODE
+        {
+          access_token: "ACCESS_TOKEN",
+          livemode: false,
+          refresh_token: "REFRESH_TOEKN",
+          token_type: "bearer",
+          stripe_publishable_key: "PUBLISHABLE_KEY",
+          stripe_user_id: "acc_stripe_user_id",
+          scope: "express",
+        }.to_json
+      else
+        status 400
+        {
+          error: "invalid_grant",
+          error_description: "Authorization code does not exist: #{code}",
+        }.to_json
+      end
+    end
+
+    get "/express/oauth/authorize" do
+      uri = URI(params["redirect_uri"])
```

Comment:
This line uses a string, the next uses a symbol. Is there a reason for these two?

Reply:
Nope, I will change it to a symbol